<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_r_alllco</class>
 <widget class="QDialog" name="dlg_r_alllco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_set">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>指定负责人</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sign">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_001">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_002">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>语言资质证明</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>理由</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选为翻译者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_endselect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择翻译者结束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_re">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>指定审核人</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
